uaclient/jobs/update_messaging.py

```diff
@@ -352,3 +357,21 @@ def update_apt_and_motd_messages(cfg: config.UAConfig) -> bool:
     # Now that we've setup/cleanedup templates render them with apt-hook
     util.subp(["/usr/lib/ubuntu-advantage/apt-esm-hook", "process-templates"])
     return True
+
+
+def refresh_motd():
```

Comment:
We might be able to just use `systemctl start motd-news.service`, which is the systemd service that get's run periodically by a timer to update the motd. That might be more reliable than looking for these commands but idk

Reply:
I wasn't able to completely remove the `update-notifier` command, since restarting the service doesn't run it apparently. But I replaced the `run-parts` one. Thanks for the suggestion @orndorffgrant

Reply:
Looking closer now - why do we need to run the update-notifier script? It should run after any `apt update` (see /etc/apt/apt.conf.d/99update-notifier), so it shouldn't get out of date like our messages could, right?